chartist-plugin-legend.js

```diff
@@ -36,6 +42,18 @@
             return a - b;
         }
 
+        // Parse input position string to enum
```

Comment:
Discussed this with an other team member, and we don't think using an enum is worth it. Allowing only exactly `"bottom"` or `"top"` (also no lower-case conversion) would be the most clear option.